pkgs/development/libraries/gpgme/default.nix

```diff
@@ -49,6 +49,15 @@ stdenv.mkDerivation rec {
 
   postPatch =''
     substituteInPlace ./configure --replace /usr/bin/file ${file}/bin/file
+  ''
+  # Apply only on Darwin to save rebuilds on Linux
+     + lib.optionalString stdenv.isDarwin ''
+    sed -i "s/LD_LIBRARY_PATH/@shlibpath_var@/" lang/python/tests/Makefile.in
+    sed -i "s/ac_subst_vars='/ac_subst_vars='shlibpath_var\n/" configure
```

Comment:
It would be nice to also send this one upstream: https://dev.gnupg.org/

Reply:
cc @AndreHeinecke

Reply:
I sent it.

Reply:
Sounds good. Can you provide a link?

Reply:
They accept patches by email, and, for some reason, it hasn’t appeared in https://lists.gnupg.org/pipermail/gnupg-devel/2020-April/ yet.

Reply:
https://lists.gnupg.org/pipermail/gnupg-devel/2020-April/034591.html